dspace-api/src/main/java/org/dspace/identifier/DOIIdentifierProvider.java

```diff
@@ -307,23 +307,12 @@ public void registerOnline(Context context, DSpaceObject dso, String identifier)
                     + "is marked as DELETED.", DOIIdentifierException.DOI_IS_DELETED);
         }
         
-        // register DOI Online
```

Comment:
Do we need the same change in the VersionedDOIIdentifierProvider?

Reply:
The `registerOnline` method is not overridden in the `VersionedDOIIdentifierProvider` - so this fix should work for the `VersionedDOIIdentifierProvider`, too.

Reply:
@kshepherd I looked at the `VersionedDOIIdentifierProvider` and @AlexanderS comment looks accurate.